src/index.js

```diff
@@ -364,10 +363,6 @@ function Datastore(options) {
     return new Datastore(options);
   }
 
-  options = common.util.normalizeArguments(this, options, {
```

Comment:
Normalization of arguments is no longer necessary?

Reply:
I believe this was an artifact of the big `google-cloud` npm module.  @stephenplusplus to verify :)

Reply:
Yep, that's right. All this did was extend the global configuration:

```js
var gcloud = require('gcloud')({ projectId: 'a' })

var datastore = gcloud.datastore()
// datastore.projectId = 'a'

var gcs = gcloud.storage({ projectId: 'b' })
// gcs.projectId = 'b'
```